lemur/common/celery.py

```diff
@@ -255,3 +258,37 @@ def sync_source(source):
     sync([source])
     log_data["message"] = "Done syncing source"
     current_app.logger.debug(log_data)
+
+
+@celery.task()
+def sync_source_destination():
+    """
+    This celery task will sync destination and source, to make sure all new destinations are also present as source.
+    Some destinations do not qualify as sources, and hence should be excluded from being added as sources
+    We identify qualified destinations based on the sync_as_source attributed of the plugin.
+    The destination sync_as_source_name reviels the name of the suitable source-plugin.
+    We rely on account numbers to avoid duplicates.
+    """
+    current_app.logger.debug("Syncing source and destination")
+
+    # a set of all accounts numbers available as sources
+    src_accounts = set()
+    sources = validate_sources("all")
+    for src in sources:
+        src_accounts.add(IPlugin.get_option('accountNumber', src.options))
+
+    for dst in destinations_service.get_all():
```

Comment:
Do we want to add a restriction somewhere  here to only check aws destinations? your call

Reply:
yeap, we do this via `destination_plugin.sync_as_source`, since we wanted to avoid hard-coding " aws destinations"